storage/innobase/include/sync0policy.ic

```diff
@@ -80,7 +80,7 @@ void MutexDebug<Mutex>::locked(
 	UNIV_NOTHROW
 {
 	ut_ad(!is_owned());
-	ut_ad(m_context.m_thread_id == os_thread_id_t(ULINT_UNDEFINED));
+	ut_ad(get_thread_id() == os_thread_id_t(ULINT_UNDEFINED));
```

Comment:
This one is most probably not needed too, because we have mutex locked and ```m_thread_id``` can only be updated under mutex.